### PR TITLE
Feature/connect ride details

### PR DESCRIPTION
**Pull request by PaigeVannelli**
## Is this a fix or a feature?
Feature

## What is the change?
Makes the profile view dynamic and allows a user to click on their profile to see their details and will allow the user to click on each matched routed to see route details.
Fix the map styles to show the radius around the origin and destination on the maps for routes. 

## What does it fix?
Allows us to used fetched data from the backend for each route. 

## Where should the reviewer start?
Review Map Detailed Route and App somponent. 

## How should this be tested?
n/a

## Relevant ScreenShots
n/a

### CODE REVIEW

File: src/components/Map/Map.tsx [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
-  }
+    initialCenter={latLong}
+    >
+       <Circle
+        radius={250}
+        center={latLong}
+        onMouseover={() => console.log('mouseover')}
+        onClick={() => console.log('click')}
+        onMouseout={() => console.log('mouseout')}
+        strokeColor='transparent'
```
Comment (itsnameissteven) [inline src/components/Map/Map.tsx:30]: are these console logs just for testing purposes?

File: src/components/DetailedRoute/DetailedRoute.tsx [hunk 2]
```diff
@@ -1,27 +1,69 @@
...
+        .catch(error => console.error('Error', error));
+      })
   }, [userId])
 
-  const days = detailedRouteData.route.days.map((day, index) => {
+  const days = matchedUser?.ride_days.map((day, index) => {
     return (
       <div className='day' key={index}>
-        {(day === 'tuesday' || day === 'thursday') ? day.charAt(0).toUpperCase() + day.slice(1,2) : day.substring(0, 1).toUpperCase()}
+        {(day === 'tuesday' || day === 'thursday' || day === 'saturday' || day === 'sunday') ?
```
Comment (alexthompson207) [inline src/components/DetailedRoute/DetailedRoute.tsx:64]: OoOo nice job switching this to cover Saturday and Sunday!

File: src/components/DetailedRoute/DetailedRoute.tsx [hunk 3]
```diff
@@ -36,39 +78,49 @@ const DetailedRoute: FC<DetailedRouteProps> = ({ userId }) => {
...
       </section>
+      {!currentUser ?
       <button className="registration__button btn">
         Request a Hitch
       </button>
+      :
+      <button className="registration__button btn">
+        Add a Route
+      </button>
+      }
```
Comment (alexthompson207) [inline src/components/DetailedRoute/DetailedRoute.tsx:123]: This looks great!

**Review by itsnameissteven — APPROVED** [review]
This looks really good. Thanks for making these components dynamic. I cant wait to see it hooked up to the backend.

**Review by alexthompson207 — APPROVED** [review]
Really nice job with this component, the radius map feature is so cool! Great work!!

**Pull request merged**